docs/concepts/workloads/controllers/deployment.md

```diff
@@ -122,18 +122,18 @@ To see the ReplicaSet (`rs`) created by the deployment, run `kubectl get rs`:
 
 ```shell
 NAME                          DESIRED   CURRENT   READY   AGE
-nginx-deployment-2035384211   3         3         3       18s
+nginx-deployment-6c54bd5869   3         3         3       15s
 ```
 
-Notice that the name of the ReplicaSet is always formatted as `[DEPLOYMENT-NAME]-[POD-TEMPLATE-HASH-VALUE]`. The hash value is automatically generated when the Deployment is created.
+Notice that the name of the ReplicaSet is always formatted as `[DEPLOYMENT-NAME]-[ENCODED POD-TEMPLATE-HASH-VALUE]` (the format is `[DEPLOYMENT-NAME]-[POD-TEMPLATE-HASH-VALUE]` for Kubernetes v1.7 and before). The hash value is automatically generated when the Deployment is created.
```

Comment:
How this is rendering in the docs is actually hard to read.  And the information regarding v1.7 and before would best be documented using our callout style guidelines.  Please see https://kubernetes.io/docs/home/contribute/style-guide/#callout-formatting  for how to do this.